lib/model/graph.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * graph.js
+ * @author John O'Brien
+ *
+ * Definition for the Graph class. The Graph class holds on to every node
+ * and connection in the entire graph and has several helper methods
+ * for manipulating them..
```

Comment:
Sorry, that was a generic comment I put there when I was starting to write it. I had assumed their would be more methods necessary to make graphs work.